nucypher/blockchain/eth/sol/source/contracts/WorkLock.sol

```diff
@@ -144,4 +156,16 @@ contract WorkLock {
         return remainingWork.sub(completedWork);
     }
 
+    /**
+    * @notice Burn unclaimed tokens
+    **/
+    function burnRemaining() public {
+        require(block.timestamp >= endBidDate, "Burning tokens allowed when bidding is over");
+        require(remainingTokens > 0, "There are no tokens that can be burned");
+        token.approve(address(escrow), remainingTokens);
+        escrow.burn(remainingTokens);
+        emit Burnt(msg.sender, remainingTokens);
```

Comment:
Is there any practical difference in outcome between actively burning your Nu and not claiming them?

Reply:
For now if staker does not claim Nu (in worklock) then can do it latter